low-level/framework/utils/salt_util.py

```diff
@@ -19,7 +19,7 @@
 """
 
 import configparser
-from cortx.utils.kvstore.pillar import PillarStorage
+from cortx.utils.kv_store.kv_store_collection import PillarStore as PillarStorage
```

Comment:
Why this renaming required?

Reply:
In chat threads this has details, kvstore class was removed, as it was some duplicate class & was not supposed to be used as per Ujjwal. I dont see any communication for that though

Reply:
In that case, we have to change our usage to `PillarStore`

Reply:
PR #298 has this change and merged into main. So I reverted this now.
After rebasing with main, all good here.